concepts/0003-protocols/roles-participants-etc.md

```diff
@@ -1,83 +1,41 @@
-# Roles, Participants, Parties, and Controllers
+### Roles, Participants, Parties, and Controllers
 
-## Roles
+#### Roles
 
-The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take on an
-interaction. 
+The __roles__ in a protocol are the perspectives (responsibilities, privileges) that parties take i an
+interaction.
 
 This perspective is manifested in three general ways:
 
  * by the expectations that a party takes on in a protocol (ex. a role may be expected to do something to start a protocol).
  * by the messages that a party can and does use in the course of the protocol (some messages may be reserved for a single role, while other may used by some if not all roles).
  * by the state and the transition rules
- 
-Like parties, roles are normally known at the start of the protocol but this is not a requirement.
 
-In an auction protocol, there are only two roles--*auctioneer*
-and *bidder*--even though there may be many parties involved.
```

Comment:
I don't like losing this example.

Reply:
I don't think this example contributes to the understanding, because such a protocol introduces a host of other issues not accounted for in these protocols. A casual reader would either think this is inherently supported, or go into a rabbit hole of trying to figure out how to make an "auction" protocol work. IMHO, neither result is helpful in getting the basics of protocols.

Reply:
Right. The intent of the comment is to suggest the host of other issues you're talking about. That's part of the scope of protocols. That's what we're trying to teach here -- that simplistic models don't account for all the complexity we're trying to account for.

Reply:
Reverted